Kernel/Devices/IDEDiskDevice.h

```diff
@@ -29,7 +42,7 @@ class IDEDiskDevice final : public IRQHandler
     virtual bool write_blocks(unsigned index, u16 count, const u8*) override;
 
 protected:
-    IDEDiskDevice();
+    IDEDiskDevice(DriveType type);
```

Comment:
Let's use `explicit` to prevent accidental type conversions.
Also, in Serenity we omit redundant parameter names in declarations (where they are not adding any information that's not already in the parameter type.)

```suggestion
    explicit IDEDiskDevice(DriveType);
```